episodes/02-importing-data.md

```diff
@@ -60,6 +60,10 @@ To import the data for the exercise below, follow the instructions in [Setup](ht
 7. From OpenRefine 3.4 onwards there is an option to Trim leading \& trailing whitespace from strings when importing separator-based files. Keeping this checked will ensure that values like `English` and `English `, which differ by a single trailing space, are not treated as different values after the import
 8. Make sure the `Attempt to parse cell text into numbers` box is not checked, so OpenRefine doesn't try to automatically detect numbers because this could cause errors such as confusion between date formats (e.g. DD/MM/YYYY vs MM/DD/YYYY).
 9. The Project Name box in the upper right corner will default to the title of your imported file. Click in the `Project Name` box to give your project a different name, if desired.
+:::::::::::::: instructor
+This is a good moment to review the points from [What Should I Know When Working with OpenRefine?](../01-introduction.md#what-should-i-know-when-working-with-openrefine)
```

Comment:
This addresses @ostephens comment. It also corrects the linking by making it relative to the current file. 

```suggestion
:::::::::::::: instructor

This is a good moment to review the points from [What Should I Know When Working with OpenRefine?](01-introduction.md#what-should-i-know-when-working-with-openrefine)
```

Reply:
Oh! Excellent.  Thank you, Zhian!  Should I add a pr in the https://carpentries.github.io/sandpaper-docs/episodes.html#links about when to use "../" at the beginning?  I'm not sure I'd know how far to take that or what quite to say.

Reply:
Yes, please! It would definitely help to get the right messaging across.

Reply:
Additionally, I think I've found a bug in {sandpaper} (or maybe feature of pandoc) as I cannot seem to get the instructor note embedded in this list inside this callout to display properly.

Reply:
That seemed to work OK for me locally @zkamvar when I added the blank line 
<img width="740" alt="Screenshot 2023-06-30 at 15 48 40" src="https://github.com/LibraryCarpentry/lc-open-refine/assets/576174/45831926-94a4-4549-8b34-5998955add9c">

Reply:
Unless you are expecting a different display that I'm missing?

Reply:
Formatting callout blocks within list items is tricky at best because you have to make sure to indent it with the rest of the list and give it plenty of room and make sure you have: 

```markdown
1. short list
2. long list item
   with wrapping
3. long list item with four spaces for a newline    
   ::::::::::::: instructor
   
   with an instructor note
   
   ::::::::::::::::::::::::
4. short list item
```

The problem with this is that even after formatting it like above, it still appears like this: 

![screenshot of list item where instructor note is still formatted as text](https://github.com/LibraryCarpentry/lc-open-refine/assets/3639446/c7d0038a-160e-436d-812f-2f633346b1af)

My suggestion would be to move this to the top of the list or add extra padding before and after the note block to separate it from the list (though that would effectively create two lists, but the second list starts at 10).

Reply:
There needs to be a blank line before `:::: instructor` for it to render

Reply:
@ostephens, you are correct! From https://pandoc.org/MANUAL.html#block-content-in-list-items:

> A list item may contain multiple paragraphs and other block-level content. However, subsequent paragraphs must be preceded by a blank line and indented to line up with the first non-space content after the list marker.

```markdown
1. short list
2. long list item
   with wrapping
3. long list item with four spaces for a newline
   
   ::::::::::::: instructor
   
   with an instructor note
   
   ::::::::::::::::::::::::
5. short list item
```